### PR TITLE
Putting first example bites back as per request, but added some twists

### PR DESCRIPTION
**Pull request by bbelderbos**

### CODE REVIEW

File: fibonnaci-sequence/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+fn fibonacci(n: i32) -> u32 {
+    match n {
+        n if n < 0 => panic!("Negative input is not allowed"), // Handle negative input by panicking
```
Comment (bbelderbos) [inline fibonnaci-sequence/src/lib.rs:3]: added this to make it a bit more complicated, hence medium

File: reverse-string/exercise.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+Level (easy, medium, hard): easy
+Tags (2-5): strings, functions, iterators
+GitHub user to credit: pybites
+---
+
+# Reverse a String
```
Comment (bbelderbos) [inline reverse-string/exercise.md:7]: good as is, bite 8 on codechalleng.es

File: temperature-convert-and-even-number/exercise.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+Level (easy, medium, hard): easy
+Tags (2-5): functions, math, basic-rust
+GitHub user to credit: pybites
+---
+
+# Temperature Conversion and Even Check
```
Comment (bbelderbos) [inline temperature-convert-and-even-number/exercise.md:7]: these were quite too simple / small scoped so I combined them.

Comment (bbelderbos) [reply temperature-convert-and-even-number/exercise.md:7]: I like shorter names for bites though, maybe we call this Simple
Calculations?

Comment (Tanner-Ray-Martin) [reply temperature-convert-and-even-number/exercise.md:7]: i like the name Simple Calculations

Comment (bbelderbos) [reply temperature-convert-and-even-number/exercise.md]: done

**Review by Tanner-Ray-Martin — APPROVED** [review]
nice! more bites

**Comment by bbelderbos** [comment]
Thanks for reviewing!

**Comment by jsh** [comment]
Thanks for putting these back! I enjoyed doing them and they gave me a good way to work through some basic setup/environment issues. As K&R says, there's a lot to be learned just getting "hello, world" to compile and run. :-)

**Pull request merged**